src/components/erc20/marketplace/wallet_balance.tsx

```diff
@@ -277,6 +285,32 @@ class WalletBalance extends React.Component<Props, State> {
             );
         }
 
+        if (web3State === Web3State.Error) {
+            content = (
+                <WalletErrorContainer>
+                    <ErrorCardStyled
+                        cursor={'default'}
+                        fontSize={FontSize.Large}
+                        icon={ErrorIcons.Warning}
+                        text={errorsWallet.mmWrongNetwork}
+                        textAlign="center"
+                    />
+                    <WalletErrorFiller top="0" left="0">
+                        {fillerBig()}
+                    </WalletErrorFiller>
+                    <WalletErrorFiller top="0" right="0">
+                        {fillerBig()}
+                    </WalletErrorFiller>
+                    <WalletErrorFiller bottom="0" left="0">
+                        {fillerSmall()}
+                    </WalletErrorFiller>
+                    <WalletErrorFiller bottom="0" right="0">
+                        {fillerSmall()}
+                    </WalletErrorFiller>
```

Comment:
Q: Why is this component repeated in this way?

Reply:
Those are four little rectangles that serve not other purpose than to be some kind of placeholder / filler in fixed positions.